cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/Cargo.toml

```diff
@@ -22,6 +22,7 @@ scale-info = { version = "2.11.1", default-features = false, features = [
 	"derive",
 ] }
 serde = { optional = true, features = ["derive"], workspace = true, default-features = true }
+tuplex = { version = "0.1", default-features = false }
```

Comment:
aha, ok, I am not sure if it is possible to hide/re-export `tuplex` inside that macro, I think we did something similar for other macros for paste?
```
// Re-export macro to avoid include paste dependency everywhere
pub use sp_runtime::paste;
```
but also ok to leave it as it is